forte/data/data_store.py

```diff
@@ -191,7 +168,7 @@ def __init__(self, onto_file_path: Optional[str] = None):
                 ...
             ]
         """
-        self.__elements: List = []
+        self.__elements: dict = {}
```

Comment:
I think we also need to update the description for elements.

Reply:
yeah, let's update the description